src/diffusers/models/unet_2d_condition.py

```diff
@@ -96,6 +96,8 @@ class UNet2DConditionModel(ModelMixin, ConfigMixin, UNet2DConditionLoadersMixin)
         norm_eps (`float`, *optional*, defaults to 1e-5): The epsilon to use for the normalization.
         cross_attention_dim (`int` or `Tuple[int]`, *optional*, defaults to 1280):
             The dimension of the cross attention features.
+        num_transformer_blocks (`int` or `Tuple[int]`, *optional*, defaults to 1):
+            The number of transformer blocks of type [`~models.attention.BasicTransformerBlock`]. Only relevant for [`~models.unet_2d_blocks.CrossAttnDownBlock2D`], [`~models.unet_2d_blocks.CrossAttnUpBlock2D`], [`~models.unet_2d_blocks.UNetMidBlock2DCrossAttn`].
```

Comment:
So a Transformer block can be a UNet block? I don't find the `num_transformer_blocks` name to be a good one to encompass all the blocks we're supporting here. But cannot think of a better one, either. So, okay to ignore I guess.

Reply:
Yeah good point, maybe `transformer_layers_per_block` is better?

Reply:
Better for sure.